crates/net/network/src/peers.rs

```diff
@@ -997,8 +1031,10 @@ impl ConnectionInfo {
 /// Tracks info about a single peer.
 #[derive(Debug, Clone)]
 pub struct Peer {
-    /// Where to reach the peer
-    addr: SocketAddr,
+    /// Peer's TCP address.
+    tcp_addr: SocketAddr,
+    /// Peer's UDP address if it's port is different to the TCP's.
```

Comment:
hmm we don't store this conditionally, so the comment is wrong?

Reply:
the udp is option, i think you're looking at the upper line.